e2e/testgames/gamebenchmark/main.go

```diff
@@ -123,7 +71,31 @@ func main() {
 		panic(eris.ToString(err, true))
 	}
 
-	initsystems, systems = initializeSystems(testFlag, testKeys, initsystems, systems)
+	// TODO: Unused decorator somehow this breaks profiling (it ends up with missing pieces of the stack).
+	// TODO: Figure out how to get this to work.
```

Comment:
I think at some point previously we had decided to remove uncommented code and put it in a Linear ticket for the task to deal with that feature? Can we do that here?

Reply:
sure.